dimreduction.go

```diff
@@ -186,7 +186,9 @@ func (p *PCA) Transform(m mat.Matrix) (mat.Matrix, error) {
 
 	//var proj mat.Dense
 	var proj sparse.CSR
-	proj.Mul(m.T(), p.pc.VectorsTo(nil).Slice(0, r, 0, p.K))
+	dst := mat.Dense{}
```

Comment:
Could you please use the form:

```
var dst mat.Dense
```

Reply:
Do you want it grouped with the `proj` declaration?

```suggestion
	var (
	    proj sparse.CSR
	    dst  mst.Dense
	)
```

Reply:
I've gone ahead and committed the changes. I left the variables ungrouped to be consistent with your other declarations.